test/schemas/comment_test.fbs

```diff
@@ -0,0 +1,15 @@
+// Line 1 comment
+
```

Comment:
Fails with this kind of comments:
```
//
// asdf
//
```

Error:
```
iex(5)> Eflatbuffers.Schema.parse!(File.read!("test/lilli-proto/ds_core/overrideevent.fbs"))
** (MatchError) no match of right hand side value: {:error, {3, :schema_lexer, {:illegal, ~c"//\n"}}, 3}
    (eflatbuffers 0.1.0) lib/eflatbuffers/schema.ex:41: Eflatbuffers.Schema.lexer/1
    (eflatbuffers 0.1.0) lib/eflatbuffers/schema.ex:28: Eflatbuffers.Schema.parse/1
    (eflatbuffers 0.1.0) lib/eflatbuffers/schema.ex:18: Eflatbuffers.Schema.parse!/1
    iex:5: (file)
```

Reply:
added some immediate mixes and merging for now. all side-effects to be address while testing our set of contracts.